PilotDataPlatform/arranger
# PILOT-4624: Add nextflow arranger server filter

## Summary

- Add arranger server filter for nextflow workflow runs (`nf-runs` index)
- Remove reference to `indoctestproject` in storybook
- Update storybook to call admin api independently from nextflow server filter to ensure we don't call the server filter when selecting the arranger project to attach to storybook

**Note:** When running storybook, it has defaults for arranger API's (admin, server filter) and elasticsearch instance. These will need to be set during deployment: https://github.com/PilotDataPlatform/arranger/blob/patched-version/modules/components/src/utils/config.js#L18-L19

## JIRA Issues

https://indocconsortium.atlassian.net/browse/PILOT-4624

## Type of Change


## Testing

Are there any new or updated tests to validate the changes?


## Test Directions
Ensure you are using node js version 14.21.0 (`nvm use 14.21.0`)
1. Run nextflow arranger server filter: `cd modules/nf-server-filter; npm install; node server.js`
2. Run nextflow admin server: `cd modules/admin; ; npm install; node admin-server.js`
3. Launch storybook: 
     - `cd modules/mapping-utils; npm install`
     - from arranger root directory: `npm install; npm run dashboard`
